azerothcore/azerothcore-wotlk
# fix(DB/Vendor)Zorbin Fandazzle's vendor gossip is now unavaliable if required quests not rewarded

## Changes Proposed:
-  Make Zorbin Fandazzle's vendor gossip unavaliable if quests  Zapped Giants and Fuel for the Zapping are not rewarded.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/10344
- Closes https://github.com/chromiecraft/chromiecraft/issues/2862

## SOURCE:

Same as the issue
## Tests Performed:

- 
- 


## How to Test the Changes:



Go to Feralas
Find Zorbin on the shore
Check his best deals, anywhere!
1. Apply
2. Get a character that didn't do required quests (Any below level 45 will do)
3. .go creature id 14637
4. Check if the " Got anything interesting for sale, Zorbin?" gossip is in there.

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
